addon/tailwind/components/bourbon-button.css

```diff
@@ -17,81 +17,81 @@
   @apply .bourbon-border-transparent;
 }
 
-.bourbon-button--icon-only {
+.BourbonButton-icon {
   @apply .bourbon-p-2;
   @apply .bourbon-min-w-button-icon;
   @apply .bourbon-min-h-button-icon;
   @apply .bourbon-border-mercury;
 }
 
-.bourbon-button--icon-only svg {
+.BourbonButton-icon svg {
   width: 20px;
   height: 20px;
 }
 
-.bourbon-button--icon-text svg {
+.BourbonButton-iconText svg {
   width: 16px;
   height: 16px;
 }
 
-.bourbon-primary-button {
+.BourbonButton--primary {
   @apply .bourbon-text-white;
   @apply .bourbon-bg-fern;
   @apply .bourbon-border-fern;
 }
 
-.bourbon-delete-button {
+.BourbonButton--delete {
```

Comment:
This rule should probably be after the `.BourbonButton--primary`'s hover/active/focus rules.